bindings/python/test/test_arrow.py

```diff
@@ -384,3 +389,125 @@ def test_find_decimal128(self):
         )
         table = find_arrow_all(coll, {}, schema=schema)
         self.assertEqual(table, expected)
+
+
+class TestNulls(unittest.TestCase):
+    @classmethod
+    def setUpClass(
+        cls,
+        find_fn=find_arrow_all,
+        equal_fn=unittest.TestCase.assertEqual,
+        table_from_dict=pyarrow.Table.from_pydict,
+    ):
+        if not client_context.connected:
+            raise unittest.SkipTest("cannot connect to MongoDB")
+        cls.cmd_listener = AllowListEventListener("find", "aggregate")
+        cls.getmore_listener = AllowListEventListener("getMore")
+        cls.client = client_context.get_client(
+            event_listeners=[cls.getmore_listener, cls.cmd_listener]
+        )
+
+        cls.oids = [ObjectId() for _ in range(4)]
+        cls.coll = cls.client.pymongoarrow_test.get_collection(
+            "test", write_concern=WriteConcern(w="majority")
+        )
+
+        cls.find_fn = lambda s, a1, a2, schema=None: find_fn(a1, a2, schema=schema)
+        cls.equal_fn = equal_fn
+        cls.table_from_dict = lambda s, d: table_from_dict(d)
```

Comment:
Rather than injecting these methods via setUpClass, what do you think about using methods, like:
```python
class TestNulls(unittest.TestCase):
...
    def find(self, ...):
        return find_arrow_all(...)
```

Then the other subclasses can override the method and it would make code navigation easier.